pygen/pygen_src/riscv_instr_stream.py

```diff
@@ -165,6 +165,10 @@ def __init__(self):
         self.allowed_instr = []
         self.category_dist = []
 
+    @vsc.constraint
+    def avail_reg_c(self):
+        self.avail_regs.size == 10
+

```

Comment:
Added the constraint for avail_regs in base stream class.
PyVSC expects a size constraint for avail_regs as it is defined as randsz. Hence It was throwing the error for other extensions.
The size has to be 10, otherwise, there will be constraint failure while running riscv_int_numeric_corner_stream directed case.